cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js

```diff
@@ -401,6 +401,47 @@ function constructSpannerSource(artifactsList, spannerInfo) {
   };
 }
 
+function constructAdlsSource(artifactsList, adlsInfo) {
+  if (!adlsInfo) {
+    return null;
+  }
+
+  let batchArtifact = find(artifactsList, { name: 'adls-plugins' });
+  if (!batchArtifact) {
+    return T.translate(`${PREFIX}.adls`);
+  }
+
+  batchArtifact.version = '[0.9.2, 9.0.0)';
```

Comment:
This is not needed if the backend(wrangler connection specification) is passing in the artifact information

Reply:
Since, backend is not passing this version information, this will be needed here as we can see in other connection services as well.

Reply:
By backend if you mean the wrangler service then yeah it won't sent as response but we fetch the list of artifacts from the artifacts endpoint and that should definitely have this. Can you try removing this version and trying again? I tried and it seemed to work fine.

Reply:
Agree the list artifacts api fetches artifact version but why does the other connections have this placeholder

[https://github.com/avantika3667/cdap/blob/d6a281f1f19d07c601fdd85ca8df7b04fb13d2a8/cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js#L258](https://github.com/avantika3667/cdap/blob/d6a281f1f19d07c601fdd85ca8df7b04fb13d2a8/cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js#L258)

Reply:
Avantika: It was added to pick the latest artifact version when there are multiple version of the plugin artifact present. By specifying the version range when users go to pipeline, we make the call to get the latest version from backend.

There are two ways to handle this,
1. Either we get the highest version from the pipeline config helper 
2. Or, we let the pipeline studio show the upgrade modal to upgrade to latest version

Showing #2 might be weird when we construct the pipeline by ourselves. We will have to pick the latest from the list of, say adls artifacts we have. I think that might not fall under the scope of this PR.

For now we can remove this and open a separate PR to fix the version hardcodings across all connection sources. cc: @elfenheart